### DISCUSSION TITLE
Is jodit able to use JavaScript?

### DISCUSSION THREAD

**Discussion by ivanoXD** [xdan/jodit#1165] [Q&A] [answered]
I am trying to use Jodit inside of a project and every time I use javascript I get a message that says "illegalscript"

**Comment by xdan** [comment, marked as answer]
That's Jodit stripping `<script>` from the content on purpose — it removes script tags and inline event handlers as XSS protection, so you can't run JS from inside the editor content.

If you genuinely need `<script>` to survive (e.g. a CMS where you trust the input), you can loosen the sanitizer via `cleanHTML.allowTags`, but know that you're turning off an XSS guard. What are you actually trying to do with the script? There's usually a safer route (a plugin or a placeholder you expand on render) than embedding raw JS.
